docker-compose-integration.yml

```
@@ -1,21 +1,22 @@
 version: '2.3'
 services:
   vault-svc:
-    image: "vault:${SVC_VAULT_VERSION:-0.10.0}"
+    image: "vault:${SVC_VAULT_VERSION:-1.12.1}"
+    healthcheck:
+      test: wget http://127.0.0.1:8200/v1/sys/health
+      interval: 2s
+      timeout: 10s
+      retries: 5
     cap_add:
     - IPC_LOCK
-    healthcheck:
-       test: ["CMD", "vault", "status"]
-       interval: 30s
-       timeout: 10s
-       retries: 5
     environment:
       - VAULT_DEV_ROOT_TOKEN_ID=${VAULT_DEV_ROOT_TOKEN_ID}
```

this is the same as:

```suggestion
      - VAULT_DEV_ROOT_TOKEN_ID
```